components/brave_wallet_ui/common/hooks/transaction-parser.test.tsx

```diff
@@ -110,20 +137,19 @@ describe('useTransactionParser hook', () => {
       ['0x Swap', BraveWallet.TransactionType.Other]
     ])('%s', (name, txType) => {
       it('should always be undefined', () => {
-        const { result: { current: transactionParser } } = renderHook(() => useTransactionParser(
-          mockNetwork, [], [], [], []
-        ))
+        const { result: { current: transactionParser } } = renderHook(() => useTransactionParser(mockNetwork), renderHookOptionsWithCustomStore(store))
 
         const mockTransactionInfo = getMockedTransactionInfo()
         const parsedTransaction = transactionParser({
           ...mockTransactionInfo,
           txType,
           fromAddress: '0xdeadbeef',
           txDataUnion: {
+            ...mockTransactionInfo.txDataUnion,
             ethTxData1559: {
-              ...mockTransactionInfo.txDataUnion.ethTxData1559,
+              ...mockTransactionInfo.txDataUnion.ethTxData1559!,
```

Comment:
Nice one with the `!` - didn't know about it.

Reply:
I try to avoid it in prod code, but it's fine for tests